### PR TITLE
Feature11/websockets

### PR DESCRIPTION
**Pull request by nas156**

### CODE REVIEW

File: backend/core/src/main/resources/application.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=postgres
-spring.datasource.password=admin
+spring.datasource.password=1111
```
Comment (AndersenGans) [inline backend/core/src/main/resources/application.properties]: do not commit such a thing

File: frontend/src/components/Header/index.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import Logo from './logo';
 import { NavLink } from 'react-router-dom';
-import UserElement from './UserElement';
+import UserElement from './UserElement/index';
```
Comment (MaximGoncharukSW) [inline frontend/src/components/Header/index.tsx]: You don't need to put index here

File: frontend/src/components/Header/index.tsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 import PathIcon from './icons/paths';
-import LoginRegister from './LoginRegister';
+import LoginRegister from './LoginRegister/index';
```
Comment (MaximGoncharukSW) [inline frontend/src/components/Header/index.tsx]: You don't need to put index here

File: frontend/src/setupProxy.js [hunk 4]
```diff
@@ -9,6 +9,8 @@ const apiProxy = proxy('/api', {
   }
 })
 
+const wsProxy = proxy('/ws', {target:'http://localhost:5000', ws: true})
```
Comment (MaximGoncharukSW) [inline frontend/src/setupProxy.js:12]: Remove this address. Use environments instead.

**Pull request merged**